### ISSUE TITLE
Missing first letter in module

### ISSUE THREAD

**Issue by Kargathia** [carsongee/pytest-pylint#27]
When running pytest-pylint in a windows environment, all files in package 'mypackage' fail with the `No module named ypackage\mymodule\myfile.py (fatal)` error

Debugger tracks the error to pytest_pylint.py:157 (PyLintItem.\_\_init\_\_)

```
self.rel_path = fspath.strpath.replace(
    parent.fspath.strpath, '', 1
)[1:]
```

fspath is a correct LocalPath pointing to 'mypackage\mymodule\myfile.py'
self.rel_path is missing the m in 'mypackage'

Python version: 3.4.3 (32 bit)
pytest-pylint version: 0.7.0

**Comment by carsongee** [comment]
Sorry for the slight delay, do mind checking to see if #28 solves this for you?

**Comment by Kargathia** [comment]
Yes, that solved the issue. Thanks!

While reproducing the issue with the 0.7.0 version I did notice that it only occurred for paths directly in the root of a drive. 
Calling `pytest --pylint mypackage` in C:\WorkArea\MyRepo\ would work just fine, while doing the same in J:\ would generate the error. (J:\ being a subst of C:\WorkArea\MyRepo\)
It did not matter whether the drive was a subst, or a physical drive.

I'm not sure on the how or why of that one, and the issue is solved by #28 anyway. Maybe useful for future bugs.

**Comment by carsongee** [comment]
Awesome, thanks for the info and reporting the bug!  I should have a bugfix release out shortly
